soc/v2/leon3/Kconfig

```diff
@@ -2,6 +2,9 @@
 # SPDX-License-Identifier: Apache-2.0
 
 config SOC_LEON3
-	bool "A LEON3 SOC which you can configure"
+	bool
```

Comment:
I believe we still require a `Kconfig.soc` so the soc tree can be reused in sysbuild (and potential other places in addition).

Reply:
Fixed